File: Amazon.Lambda/Felipearpa.Tyche.Function/src/Response/PoolGamblerScoreResponse.fs
namespace Felipearpa.Tyche.Function.Response


type PoolGamblerScoreResponse =
    { PoolId: string
      PoolName: string
      GamblerId: string
      GamblerUsername: string
      Position: int option
      BeforePosition: int option
      Score: int option
      BeforeScore: int option }

File: Android/data/pool/src/main/java/com/felipearpa/tyche/data/pool/domain/PoolGamblerScoreResponse.kt
package com.felipearpa.tyche.data.pool.domain

import kotlinx.serialization.Serializable

@Serializable
internal data class PoolGamblerScoreResponse(
    val poolId: String,
    val poolName: String,
    val gamblerId: String,
    val gamblerUsername: String,
    val position: Int?,
    val beforePosition: Int?,
    val score: Int?,
)
